scripts/gulp/gulpfile.ts

```diff
@@ -73,13 +76,26 @@ gulp.task(
         viteApp,
         viteLaunchpad,
       ),
-
-      // And we're finally ready for electron, watching for changes in
-      // /graphql to auto-restart the server
     ),
   ),
 )
 
+gulp.task(
+  'watch',
+  gulp.series(
+    'dev:watch',
+    // And we're finally ready for electron, watching for changes in
+    // /graphql to auto-restart the server
+    async function logInfo () {
```

Comment:
If anyone is wondering why this is async, all gulp tasks need to return a `Promise` or a stream, or it's an error.